grails-app/domain/org/pih/warehouse/reporting/CycleCountProductSummary.groovy

```diff
@@ -1,12 +1,14 @@
 package org.pih.warehouse.reporting
 
+import java.time.Instant
+
 import org.pih.warehouse.core.Location
 import org.pih.warehouse.product.Product
 
 class CycleCountProductSummary implements Serializable {
 
     String cycleCountId
-    Date transactionDate
+    Instant transactionDate
```

Comment:
based on Transaction.transactionDate (which is a Date in the domain but since we're populating the view via sql query we don't interact with the Transaction domain and so can use Instant)